test/unit/pages/confirm/controller-test.js

```diff
@@ -45,9 +52,14 @@ describe('ConfirmController', function() {
     sandbox.stub(fs, 'existsSync').returns(true);
     sandbox.spy(_$state_, 'go');
     installerDataSvc.setup();
-    for (var installer of installerDataSvc.allInstallables().values()) {
-      sandbox.stub(installer, 'detectExistingInstall').resolves();
-    }
+
+    sandbox.stub(CDKInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(CygwinInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(HypervInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(JdkInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(VirtualBoxInstallWindows.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(VirtualBoxInstallDarwin.prototype, 'detectExistingInstall').resolves();
```

Comment:
I think it should be possible to do this with the original loop (which really looks nicer), it just needs to be moved below the confirmController instantiation.